llvm/test/Transforms/PGOProfile/fix_entry_count_sampled.ll

```diff
@@ -0,0 +1,28 @@
+; RUN: llvm-profdata merge %S/Inputs/fix_entry_count_sampled.proftext -o %t.profdata
+; RUN: opt < %s -passes=pgo-instr-use -pgo-test-profile-file=%t.profdata -S | FileCheck %s --check-prefix=USE
```

Comment:
Since this is the only `FileCheck` command, can we use the default check prefix?

Reply:
Certainly.  This followed the neighboring tests `fix_entry_count.ll`'s style, which is testing similar functionality.